app/chat/[id]/_components/Message.tsx

```diff
@@ -1,27 +1,25 @@
 import dayjs from "dayjs";
 
-import { CustomMessageType } from "../_types/message";
 import { cn } from "../../../_common/shadcn/utils";
 
 interface MessageProps {
   message: CustomMessageType;
+  userId: number;
```

Comment:
이 부분도 이렇게 하는 방법이 유지보수에 도움이 될까요..? 저번에 멘토님에게 조언을 받고, 전반적으로 코드 리팩토링할때 `userId`처럼 이미 만들어진 타입은 참조하는 방식으로 모두 변경해야될지 고민중이에요.
```suggestion
  userId: User["id"];
```

Reply:
좋은 방식인 것 같습니다!
코드 리팩토링 할 때 관련된 부분은 모아서 한번에 수정해봐도 좋을 것 같아요.
배포 때 에러가 발생할 것 같아서, 지금은 적어두기만 하겠습니다.